opencensus/trace/exporters/zipkin_exporter.py

```diff
@@ -211,3 +212,24 @@ def _extract_tags_from_span(attr):
             continue
         tags[attribute_key] = value
     return tags
+
+
+def _extract_annotations_from_span(span):
+    """Extract and convert time event annotations to zipkin annotations"""
+    if span.time_events is None:
+        return []
+
+    annotations = []
+    for time_event in span.time_events:
+        annotation = time_event.annotation
+        if not annotation:
+            continue
+
+        event_time = datetime.datetime.strptime(time_event.timestamp,
+                                                ISO_DATETIME_REGEX)
+        epoch_time = calendar.timegm(
```

Comment:
it would be nice in the future for us to unify time conversions for Zipkin to a helper function say
`datetimeStrToMicroseconds`
```python
def datetimeStrToMicroseconds(timestamp):
   """
  Convert a timestamp string into a microsecond value as required by Zipkin.
  It returns an int
   """
   dt = datetime.datetime.strptime(timestamp, ISO_DATETIME_REGEX)
   mUs = (calendar.timegm(dt.timetuple()) * 1e6) + dt.microsecond
   return int(round(mUs))
```
To avoid repeating this pattern. I'll file an issue to follow-up with this.

Reply:
I've filed followup issue https://github.com/census-instrumentation/opencensus-python/issues/328 PTAL.